schema/2.0.0/schema.md

```diff
@@ -643,20 +641,20 @@ Curators MUST annotate the following columns in the `var` and `raw.var` datafram
 </tbody></table>
 <br>
 
-### feature_id
+### index
 
 <table><tbody>
     <tr>
       <th>Key</th>
-      <td>feature_id (<code>var.index</code>/<code>raw.var.index</code>)</td>
+      <td>index</td>
```

Comment:
I'm afraid there is still an implied requirement that the index must be named `index`. It might be worth including a note
index (no specific name is required)

Reply:
I meant `index` in the sense of `var.index` and `raw.var.index` which are the keys even if `adata.var.index.name` points to `feature_id`.

Do you feel it would be more precise as *the pandas.DataFrame index for `var` or `raw.var`*

Reply:
Yes, I think that would help distinguish it from the other schema properties.
_I_ understood what you meant, but I'm worried that some of the R folks might see all of the key:field_name and take them to mean column name, then treat this one the same.

Reply:
Speaking of R, we should also link the R Encoding document into the schema.